### DISCUSSION TITLE
I want to give the server name an alias and display it. What should I do

### DISCUSSION THREAD

**Discussion by 2310652493** [apache/skywalking#11734] [Q&A] [answered]
I want to give the server name an alias and display it. What should I do
![image](https://github.com/apache/skywalking/assets/88769942/362aa569-7949-4979-9e5a-f7f663bc8785)

**Comment by wu-sheng** [comment, marked as answer]
No, we don't support that. This is not a management system.

**Reply by 2310652493** [reply]
Thank

**Reply by wu-sheng** [reply]
If you need a name, deploy your own DNS, and use the domain name rather than IP to access them.
